src/test/java/com/unzer/payment/business/paymenttypes/ApplepayTest.java

```diff
@@ -26,85 +26,87 @@
-	}
+    @Test
+    public void testAuthorizeApplePayType() throws HttpCommunicationException, MalformedURLException {
+        Applepay applepay = getUnzer().createPaymentType(getApplePay());
+        Authorization authorization = applepay.authorize(BigDecimal.ONE, Currency.getInstance("EUR"), new URL("https://www.meinShop.de"));
+        assertNotNull(authorization);
+        assertNotNull(authorization.getId());
+        assertEquals(Authorization.Status.SUCCESS, authorization.getStatus());
+    }
 
-	@Ignore("missing key configuration for apple pay")
-	@Test
-	public void testChargeApplePayType() throws HttpCommunicationException, MalformedURLException {
-		Applepay applepay = getUnzer().createPaymentType(getApplePay());
-		Charge charge = applepay.charge(BigDecimal.ONE, Currency.getInstance("EUR"), new URL("https://www.meinShop.de"));
-		assertNotNull(charge);
-		assertNotNull(charge.getId());
-		assertEquals(Authorization.Status.SUCCESS, charge.getStatus());
-	}
+    @Test
+    public void testAuthorizeApplePayTypeId() throws HttpCommunicationException, MalformedURLException {
+        Applepay applepay = getUnzer().createPaymentType(getApplePay());
+        Authorization authorization = getUnzer().authorize(BigDecimal.ONE, Currency.getInstance("EUR"), applepay.getId(), new URL("https://www.meinShop.de"));
+        assertNotNull(authorization);
+        assertNotNull(authorization.getId());
+        assertEquals(Authorization.Status.SUCCESS, authorization.getStatus());
+    }
 
-	@Ignore("missing key configuration for apple pay")
-	@Test
-	public void testChargeApplePayTypeId() throws HttpCommunicationException, MalformedURLException {
-		Applepay applepay = getUnzer().createPaymentType(getApplePay());
-		Charge charge = getUnzer().charge(BigDecimal.ONE, Currency.getInstance("EUR"), applepay.getId(), new URL("https://www.meinShop.de"));
-		assertNotNull(charge);
-		assertNotNull(charge.getId());
-		assertEquals(Authorization.Status.SUCCESS, charge.getStatus());
-	}
+    @Test
+    public void testChargeApplePayType() throws HttpCommunicationException, MalformedURLException {
+        Applepay applepay = getUnzer().createPaymentType(getApplePay());
+        Charge charge = applepay.charge(BigDecimal.ONE, Currency.getInstance("EUR"), new URL("https://www.meinShop.de"));
+        assertNotNull(charge);
+        assertNotNull(charge.getId());
+        assertEquals(Authorization.Status.SUCCESS, charge.getStatus());
+    }
 
+    @Test
+    public void testChargeApplePayTypeId() throws HttpCommunicationException, MalformedURLException {
+        Applepay applepay = getUnzer().createPaymentType(getApplePay());
+        Charge charge = getUnzer().charge(BigDecimal.ONE, Currency.getInstance("EUR"), applepay.getId(), new URL("https://www.meinShop.de"));
+        assertNotNull(charge);
+        assertNotNull(charge.getId());
+        assertEquals(Authorization.Status.SUCCESS, charge.getStatus());
```

Comment:
Where are status defined? In an interface or in the transaction class?

Reply:
Hi @sixer1182, this Status-Enum is inside the Transaction-Class.

Reply:
I was confused wha you were using Authorization.Status.SUCCESS
Because it is a test with charge
Transaction.Status.SUCCESS
Would be more understandable.

Reply:
So you mean `Authorization.Status.SUCCESS`?

Reply:
I have changed it in PR https://github.com/unzerdev/java-sdk/pull/28